### ISSUE TITLE
Allow reuploading openapi.yaml to an existing collection

### ISSUE THREAD

**Issue by n14s** [EsperoTech/yaade#28]
In case of changes in the openapi.yaml, I need to delete the collection and create a new one with the new openapi.yaml. 
The Edit Panel of the Collection only allows for renaming the Collection.
Let's add a feature to re-upload the openapi.yaml for an existing collection from the edit panel.

**Comment by jonrosner** [comment]
Very good idea. The only question is how do you handle manual changes that were done to the collection after importing it from an openapi file. An easy solution would be to just overwrite everything but that feels like it can lead to unexpected deletion of properties.

**Comment by n14s** [comment]
Good point, it would be very upsetting to lose all manual entries without a warning.
But it indeed is the easiest solution.
Maybe a warning when trying to upload an updated openapi.yaml could do the trick.


While the retainment of properties is not trivial, it can be a future enhancement, with this easy solution as a basis.
